docs/product-editor-development/common-tasks.md

```diff
@@ -7,18 +7,23 @@
 Here's a snippet that adds a new block to the catalog section for simple products, between the first and second fields (order 15):
 
 ```php
+<?php
+
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
+
 if ( ! function_exists( 'YOUR_PREFIX_add_block_after_categories' ) ) {
 	/**
 	 * Add a new block to the template.
 	 */
 	function YOUR_PREFIX_add_block_after_categories( BlockInterface $product_categories_field ) {
     $product_categories_field->get_parent()->add_block(
       [
-        'id'         => 'your-prefix-id',
-        'blockName'  => 'your-block-name',
-        'order'      => $product_categories_field->get_order() + 5,
+        'id'         => 'your-checkbox-id',
+        'blockName'  => 'woocommerce/product-checkbox-field',
+        'order'      => $product_categories_field->get_order() + 1,
         'attributes' => [
-          'key'   => 'value',
+          'label'   => 'Your Checkbox',
+					'value'   => 'meta_data.your_checkbox',
```

Comment:
Indentation is off.

And, attribute should be `property`. I don't think we support the `meta_data.your_checkbox` format yet, do we? We will once we switch it to use your new hook though. Might want to choose an existing boolean property for now.